tfhe/src/core_crypto/algorithms/test/lwe_keyswitch_key_generation.rs

```diff
@@ -5,7 +5,7 @@ use crate::core_crypto::commons::generators::{
 };
 use crate::core_crypto::commons::math::random::ActivatedRandomGenerator;
 
-fn test_seeded_lwe_ksk_gen_equivalence<Scalar: UnsignedTorus>(
+fn test_seeded_lwe_ksk_gen_equivalence<Scalar: UnsignedTorus + Send + Sync>(
```

Comment:
Add to Numeric trait

Reply:
Is it merged already ? If not I'll start doing that once it's merged

Reply:
No (https://github.com/zama-ai/tfhe-rs/pull/528)
But I think it would be more interesting to merge your PR first to see bench progression as decompression is slower than fourier conversion

Reply:
fine to start doing it in next PRs ? once the updated Numeric is in main ?